decidim-bulletin_board-js/src/key-ceremony/key-ceremony.js

```diff
@@ -60,6 +60,14 @@ export class KeyCeremony {
     );
   }
 
+  /**
+   * Returns the backup function of the TrusteeWrapper
+   * @returns {String}
+   */
+  backup() {
+    return this.currentTrustee.wrapper.backup()
+  }
+
```

Comment:
Should we add the restore method too, or can we wait until needing It?

Reply:
I would wait until we actually need it. 😄